### PR TITLE
Ds 3207 (merging by case)

### PR DESCRIPTION
**Pull request by JustinCCYap**

### CODE REVIEW

**Review by oliverbock — APPROVED** [review]

File: R/datasetmergingbycasewidget.R [hunk 1]
```diff
@@ -0,0 +1,490 @@
+#' @title Widget Output for Data Set Merging By Case
+#' @description Widget shown in output for \code{flipData::MergeDataSetsByCase}.
+#' @param input.data.sets.metadata An object containing metadata for the
+#'   input data set.
+#' @param merged.data.set.metadata An object containing metadata for the
+#'   merged data set.
```
[the remark this answers is not in the document]
Comment (JustinCCYap) [reply R/datasetmergingbycasewidget.R]: Improved documentation in d5b11a3

File: R/datasetmergingbycasewidget.R [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+#' @param matched.names A matrix whose rows contain the names of the input
+#'   variables used to create the merged variables. This matrix also has the
+#'   attributes "is.fuzzy.match" and "matched.by" which are matrices indicating
+#'   whether the input variable was a fuzzy match and what they were matched by,
+#'   i.e., variable name, label or value label.
```
[the remark this answers is not in the document]
Comment (JustinCCYap) [reply R/datasetmergingbycasewidget.R]: Improved documentation in d5b11a3

File: R/datasetmergingbycasewidget.R [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+#' @param merged.names A character vector of the names of the merged variables.
+#'   Also contains the attribute "renamed.variables" which contains the
+#'   merged variables that were renamed due to name conflicts and their
+#'   original names.
```
[the remark this answers is not in the document]
Comment (JustinCCYap) [reply R/datasetmergingbycasewidget.R]: Improved documentation in d5b11a3

File: R/datasetmergingbycasewidget.R [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+#' @param omitted.variable.names A list where each element contains the names of the
+#'   variables omitted from a data set.
```
[the remark this answers is not in the document]
Comment (JustinCCYap) [reply R/datasetmergingbycasewidget.R]: Improved documentation in d5b11a3

File: R/datasetmergingbycasewidget.R [hunk 5]
```diff
@@ -0,0 +1,490 @@
...
+#' @param input.value.attributes A list where each element contains the value
+#'   attributes of input categorical variables from a data set.
```
[the remark this answers is not in the document]
Comment (JustinCCYap) [reply R/datasetmergingbycasewidget.R]: Improved documentation in d5b11a3

File: R/datasetmergingbycasewidget.R [hunk 6]
```diff
@@ -0,0 +1,490 @@
...
+#' @param is.saved.to.cloud Whether the merged data set was saved to the
+#'   Displayr cloud drive.
```
[the remark this answers is not in the document]
Comment (JustinCCYap) [reply R/datasetmergingbycasewidget.R]: Improved documentation in d5b11a3

**Pull request merged**